bindings/nodejs/examples/wallet/16-destroy-alias.ts

```diff
@@ -14,20 +14,16 @@ import { getUnlockedWallet } from './common';
 async function run() {
     try {
         // Create the wallet
-        const wallet = await getUnlockedWallet();
+        const manager = await getUnlockedManager();
```

Comment:
This is wrong

Reply:
what the hell? Must somehow have happened during the merge 🤔

Reply:
Fixed